addons/mail/static/src/components/message/message.xml

```diff
@@ -61,7 +61,15 @@
                     <t t-if="!props.isSquashed">
                         <div class="o_Message_header">
                             <t t-if="message.author">
-                                <div class="o_Message_authorName o_Message_authorRedirect o_redirect" t-on-click="_onClickAuthorName" title="Open profile">
+                                <div role="link" tabindex="0" class="o_Message_authorName o_Message_authorRedirect o_redirect" t-on-click="_onClickAuthorName" title="Open profile">
+                                    <span
+                                        t-if="message.message_type === 'notification' or !(message.is_discussion or message.is_notification)"
+                                        class="sr-only"
+                                    >
+                                        <t t-if="message.message_type === 'notification'">System notification by</t>
+                                        <t t-else="">Note by</t>
+                                        <span t-esc="' '"/>
+                                    </span>
```

Comment:
I don't understand these changes, but it looks too complex and we shouldn't have change in tests.

Reply:
I'ts an invisible text (class `sr-only)` to differentiate internal notes from standard messages. Instead of just author name, the following is printed:
- Note by Mitchell Admin (in case of  internal notifications)
- Mitchell Admin (in case of standard messages)
- System notification by OdooBot

Only author name is visible, so visually there should be no changes.

This is the simplest solution I could think of, but any suggestions are welcomed.

Reply:
Done, simplified a lot, now it's just an invisible img